### ISSUE TITLE
Changes screen reader shortcut to go backwards doesn't work

### ISSUE THREAD

**Issue by misolori** [microsoft/vscode#94197]
Refs #93301

> F7 and shift+F7 navigate you through all the diffs and the screen reader correctly reads the added and removed content

`Shift + F7` does not allow me to go backwards through the diff changes and keeps going down the list. 

It's worth mentioning that i'm on a Mac using Parallels and have use `fn` to get to my function keys, so I press `fn + F7 + Shift`.

**Comment by misolori** [comment]
Also realized that when there's a single range of changes (like two edited lines next to each other) the `F7` shortcut just goes down line-by-line. This is confusing because it's different from when you have multiple ranged changes and that cycles through each range set.

**Comment by isidorn** [comment]
Shift + F7 navigation works just fine for me. Why don't you simply try to repro not in a VM? So just try to repro on your mac please. If you can repro can you please provide some example (for instance do some changes in some file in vscode repo and let me know the steps).

As for your other confusion what would you expect F7 to do then? Be a no-op?
